src/components/Banner/Banner.tsx

```diff
@@ -1,7 +1,7 @@
 import _ from 'lodash';
 import React from 'react';
 import PropTypes from 'react-peek/prop-types';
-import ReactTransitionGroup from 'react-transition-group/CSSTransitionGroup';
+import { CSSTransition, TransitionGroup } from 'react-transition-group';
```

Comment:
This was the recommended approach here: https://github.com/reactjs/react-transition-group/blob/master/Migration.md